GHzDACs/ghz_fpga_server.py

```diff
@@ -2133,17 +2152,20 @@ def adc_bringup(self, c):
     #   - ADC boards: either one waveform (averaged) for whole run
     #                 or one demodulation packet for each repetition
 
-# Runners contain information for running a sequence on a particular board
+# Runners contain information for running a sequence on a particular board.
 #
-# other info these should provide:
-# - what to upload
-# - whether upload is optional (based on setup state)
-# - how many packets to expect, based on repetitions
-# - whether discard is possible
+# Other info these should provide:
+# - what to upload.
+# - whether upload is optional (based on setup state).
+# - how many packets to expect, based on repetitions.
+# - whether discard is possible.
+
+# Some helper methods.
+def _AssertHasJumpTable(dev):
```

Comment:
we prefer lower-case names for functions, and would like to move to `lower_case_with_underscore`, as recommended by PEP8 (the `camelCase` names we use were inherited from twisted's style, which is a bit of an outlier). How about `_assert_has_jump_table`?